content/install_maintain/installation/installing-cluster.md

```diff
@@ -221,24 +184,93 @@ postgresql_server:
         # Should be the same on all nodes
         postgres:
             replicator_password: '<strong password for replication user>'
+  
```

Comment:
Adding the `monitoring` part for specifying credentials. According to @geokala comment at the presentation on Thursday. This should also be done in the RabbitMQ config file.

Reply:
well in the rabbit config we have Prometheus section with the credentials.

Reply:
These credentials are not the same. The `monitoring` credentials are used when trying to connect to the monitoring service (Prometheus application). The credentials in the `prometheus` section are used in order to configure Prometheus.

Reply:
Talked with @mateumann . We dont need to add the monitoring section to db and rabbit nodes , only to the client config.